benchmarks/src/main/scala/ai/h2o/sparkling/benchmarks/SampleBenchmark.scala

```diff
@@ -0,0 +1,41 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.benchmarks
+
+import org.apache.spark.ml.h2o.algos.H2OAlgorithm
+import org.apache.spark.sql.types.StringType
+import org.apache.spark.sql.functions._
+
+class SampleBenchmark(context: BenchmarkContext) extends BenchmarkBase(context) {
+  override protected def body(): Unit = {
+    import context.spark.implicits._
+    context.spark.range(20000000)
+      .withColumn("str", 'id.cast(StringType))
```

Comment:
I guess this PR is just infrastructure, the content of benchmarks will come in the following PR?

Reply:
Yep, This should be covered by the Jira ticket [SW-1293](https://0xdata.atlassian.net/browse/SW-1293).